viewer/packages/sector-loader/src/GltfSectorRepository.ts

```diff
@@ -55,7 +55,15 @@ export class GltfSectorRepository implements SectorRepository {
     if (this._gltfCache.has(cacheKey)) {
       return this._gltfCache.get(cacheKey);
     }
-    const consumedSector = await this._gltfSectorLoader.loadSector(sector);
+
+    const consumedSector = await this._gltfSectorLoader.loadSector(sector).catch(() => {
+      return undefined;
```

Comment:
Do we handle the return value somewhere for the `loadSector`, wanted to understand if `return undefined` or `return null` is suitable here

Reply:
undefined is returned to from the method, but from the catch lambda into the `consumedSector` variable. Which is then checked for undefined in the next line.